Source/SwiftLintFramework/Rules/ProhibitedSuperRule.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ProhibitedSuperRule.swift
+//  SwiftLint
+//
+//  Created by Aaron McTavish on 12/12/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct ProhibitedSuperRule: ConfigurationProviderRule, ASTRule, OptInRule {
```

Comment:
I had considered it, but had decided to make it opt-in as the rule is based on method signature and some of the signatures are not particularly unique. Without adding additional checks to see if the class is a descendent of the correct object, I was worried there would likely be some false positives. I thought that might be a good addition in a future PR.

But if you feel that those developers should either change the names of their methods or mark it as excluded in the configuration, I'm happy to make it opt-in. :smile_cat: